pkgs/by-name/bi/birdtray/package.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
```

Comment:
Please pass the file through `nixfmt-rfc-style`

Reply:
Same reasoning as above, people complain about reformatting.

Reply:
I dont :-)